src/main/java/io/github/mzmine/datamodel/features/types/graphicalnodes/FeatureShapeChart.java

```diff
@@ -105,23 +100,14 @@ public FeatureShapeChart(@NotNull ModularFeatureListRow row, AtomicDouble progre
       defaultRange = new Range(0, 1);
     }
 
-    setPrefHeight(GraphicalColumType.DEFAULT_GRAPHICAL_CELL_HEIGHT);
-    setPrefWidth(GraphicalColumType.DEFAULT_GRAPHICAL_CELL_WIDTH);
 
     chart.addDatasets(datasets);
     chart.getXYPlot().getDomainAxis().setRange(defaultRange);
     chart.getXYPlot().getDomainAxis().setDefaultAutoRange(defaultRange);
 
-    BufferedImage img = chart.getChart()
-        .createBufferedImage(GraphicalColumType.DEFAULT_GRAPHICAL_CELL_WIDTH,
-            GraphicalColumType.DEFAULT_GRAPHICAL_CELL_HEIGHT);
-    ImageView view = new ImageView(SwingFXUtils.toFXImage(img, null));
-    view.setOnMouseClicked(e -> MZmineCore.runLater(() -> {
-      getChildren().remove(view);
-      getChildren().add(chart);
-      e.consume();
-    }));
-
-    MZmineCore.runLater(() -> getChildren().add(view));
+    var width = GraphicalColumType.LARGE_GRAPHICAL_CELL_WIDTH;
```

Comment:
why change?

Reply:
The chromatograms can be quite wide and its better to start out with a wider column. Better to see if there is a shoulder on a peak or if there are multiple peaks in a chromatogram

Reply:
See above reply